pyproject.toml

```
@@ -11,8 +11,8 @@ pydantic = "^2.5"
 tabulate = "^0.9"
 mdutils = "^1.6"
 loguru = "^0.7"
-packaging = "^23.2"
-typer = { extras = ["all"], version = "^0.9" }
+packaging = "^24.0"
+typer = {extras = ["all"], version = "^0.12"}
```

When [installing](https://github.com/ScDor/adopt-ruff/actions/runs/9709737218/job/26799025265?pr=12#step:5:30), there's a warning that `typer 0.12.3 does not provide the extra 'all'`. 

Looks like it is [no longer needed](https://typer.tiangolo.com/release-notes/#0120) since 0.12 

```suggestion
typer = {version = "^0.12"}
```

---

Ah yes, the author changed the packaging - I'll fix that immediatly !

---

Fixed